### ISSUE TITLE
Google-Mobile-Ads-SDK 7.68.0 and xcframework don't display Class name in storyboard

### ISSUE THREAD

**Issue by SmartHbdev** [CocoaPods/CocoaPods#10484]
* [x] I've read and understood the [*CONTRIBUTING* guidelines and have done my best effort to follow](https://github.com/CocoaPods/CocoaPods/blob/master/CONTRIBUTING.md).

# Report

## What did you do?

`pod install`

## What did you expect to happen?

Use Custom class from Google-Mobile-Ads-SDK in the storyboard

## What happened instead?

Google-Mobile-Ads-SDK class don't appear in the drop down menu to select the class name.
Adding the Class name manually don't load it as expexted.
For example GADUnifiedNativeAdView don't load the outlets and we can't connect them to the views.

Google-Mobile-Ads-SDK 7.68.0 use xcframework, that's probably the issue

### Stack

```
   CocoaPods : 1.10.1
        Ruby : ruby 2.6.3p62 (2019-04-16 revision 67580) [universal.x86_64-darwin20]
    RubyGems : 3.0.3
        Host : macOS 11.2.1 (20D74)
       Xcode : 12.4 (12D4e)
         Git : git version 2.24.3 (Apple Git-128)
Ruby lib dir : /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib
Repositories : cocoapods - git - https://github.com/CocoaPods/Specs.git @ e47b0bc35104563d4cc9255d44fd347eefe96093
               trunk - CDN - https://cdn.cocoapods.org/
```

### Installation Source

```
Executable Path: /usr/local/bin/pod
```

### Plugins

```
cocoapods-deintegrate : 1.0.4
cocoapods-plugins     : 1.0.0
cocoapods-search      : 1.0.0
cocoapods-stats       : 1.1.0
cocoapods-trunk       : 1.4.1
cocoapods-try         : 1.1.0
```

### Podfile

```ruby
# Uncomment this line to define a global platform for your project
platform :ios, '9.0'
# Uncomment this line if you're using Swift
# use_frameworks!

# ignore all warnings from all pods
inhibit_all_warnings!

use_frameworks!

#source 'https://github.com/CocoaPods/Specs.git'

target 'Explote' do
  
  post_install do |installer|
    installer.pods_project.targets.each do |target|
      target.build_configurations.each do |config|
        if config.build_settings['IPHONEOS_DEPLOYMENT_TARGET'].to_f < 9.0
          config.build_settings['IPHONEOS_DEPLOYMENT_TARGET'] = '9.0'
        end
      end
    end
  end
  
  pod 'Firebase/Analytics'
  pod 'Firebase/Performance'
  
  pod 'MWFeedParser'
  pod 'CocoaLumberjack'
  pod 'Fabric'
  pod 'Crashlytics'
  pod 'Google-Mobile-Ads-SDK', '7.68.0'

  pod 'CocoaAsyncSocket'
  pod 'HHServices'
  
  pod 'Toast'
  
  target 'ExploteTests' do
    inherit! :search_paths
  end
  
  target 'ExploteUITests' do
    inherit! :search_paths
  end
  
  
  
end
```

## Project that demonstrates the issue
Any project with Google-Mobile-Ads-SDK 7.68.0 show this issue

**Comment by dnkoutso** [comment]
Please file an issue at the corresponding google repository. If its determined to be a cocoapods issue we can re-open this one and figure out what the problem is.

**Comment by SmartHbdev** [comment]
I don't think they have a public repo. About xcframework i have see many issue here so maybe you can check if it's a Cococpod issue or SDK issue ?

I think the xcframework is not linked correctly somewhere, that's why the class name don't appear in the storyboard.
But if i don't need this part and stay with code only, the project works fine.

Maybe a path issue or something like this with pod.
There is a open issue on pod (i have lost the issue number) who say something similar about xcframework and something missing with pod compared to integration without pod.

I have spend hours of searching today, please take a quick look.

**Comment by SmartHbdev** [comment]
Just add pod 'Google-Mobile-Ads-SDK', '7.68.0' in any project and see that no class that start with GAD is listed.
I can see them in the xcframework file but the storyboard don't see them.

**Comment by dnkoutso** [comment]
Going to keep open but I personally have little time to investigate this.

**Comment by paulb777** [comment]
Maybe related to #10459 and might need a similar `-ObjC` workaround.

**Comment by SmartHbdev** [comment]
> Maybe related to #10459 and might need a similar `-ObjC` workaround.

I can see -ObjC is already in other linker flag, don't know why or why not.
It's probably not the same problem.

**Comment by dnkoutso** [comment]
@paulb777 I know this is old, any idaes? Maybe 1.11.0 master has fixed issues?

**Comment by paulb777** [comment]
Yes, it might even be fixed in 1.10.2 via #10460

**Comment by dnkoutso** [comment]
We can reopen if more reports come in
